Documentation/ceph-filesystem.md

```diff
@@ -139,6 +139,44 @@ After creating it with `kubectl create -f kube-registry.yaml`, you now have a do
 #### Kernel Version Requirement
 If the Rook cluster has more than one filesystem and the application pod is scheduled to a node with kernel version older than 4.7, inconsistent results may arise since kernels older than 4.7 do not support specifying filesystem namespaces.
 
+## Consume the Shared File System: K8s PVC
+
+In order to comsume shared file system by using PVC, you need create storageclass firstly. Here is the sample of storageclass:
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rook-ceph-fs
+provisioner: ceph.rook.io/filesystem
```

Comment:
The storage class will need something to identify which ceph cluster since it's possible to run multiple ceph clusters with the same operator. For example, see the [clusterNamespace](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/storageclass.yaml#L26) property for the rbd storageclass

Reply:
@travisn add ```clusterNamespace``` would be great. thanks.